build/index.js

```diff
@@ -35,23 +38,22 @@ function getCurrentGitBranch(root) {
     // Only bother getting fancy if the root is CONTENT_ROOT.
     // For other possible roots, just leave it to the default.
     if (root === CONTENT_ROOT) {
-      if (process.env.GITHUB_REF_NAME_SLUG) {
-        name = process.env.GITHUB_REF_NAME_SLUG;
+      if (process.env.GITHUB_REF) {
+        name = process.env.GITHUB_REF.split("/").slice(2).join("/");
```

Comment:
I don't understand this change. 
According to https://github.com/FranzDiebold/github-env-vars-action I appreciate that if the branch has a `/` in it, it was actually `GITHUB_REF_NAME` you should have used.

Reply:
`rg github-env` yields nothing. We're not using that action right now are we?
https://github.com/FranzDiebold/github-env-vars-action/blob/adb7fc4ab17a7234ca657ae5ab4b15caf5ad0931/dist/index.js#L253

Reply:
Gah! I see. This one was one of the *additionally exposed* ones. Good catch!